src/schemas/pa-yaml/v3.0/pa.schema.yaml

```diff
@@ -126,10 +126,11 @@ definitions:
     $comment: Defines reusable schema for validating the pattern allowed for control type identifiers.
     type: string
     # The value of this identifier takes the format:
-    #   ( <invariantFamily> '/' )? <invariantName>
+    #   ( <invariantFamily> '/' )? <invariantName> ( '@' <version> )?
     #   Where the set of allowed characters is limited.
+    #   Where <version> is a 3-part version number (e.g. "1.20.30").
     pattern: |-
-      ^([A-Z][a-zA-Z0-9]*/)?[A-Z][a-zA-Z0-9]*$
+      ^([A-Z][a-zA-Z0-9]*/)?[A-Z][a-zA-Z0-9]*(@\d+\.\d+\.\d+)?$
```

Comment:
What is the default version when version is not specified?

Reply:
it's not specified in the schema. This change just indicates the format for the string that is expected.
In the compiler is where a default version will get applied. When no version is set, it'll imply the current version of the control.